sys/linux/test/landlock_ptrace

```diff
@@ -0,0 +1,31 @@
+# Creates independent Landlock hierarchies.
+#
+# fork() is not present on these architectures:
+# requires: threaded=0 -arch=arm64 -arch=riscv64
```

Comment:
Well… -arch=* seems to not be recognized: https://github.com/google/syzkaller/pull/2380/checks?check_run_id=1838984246#step:4:77

Reply:
The failing test does not even look at the constraints, nor it tries to execute these program, it just checks that syntax is correct:
https://github.com/google/syzkaller/blob/23a562dfb3a9986a066a1341c2cfc9e87a8fa164/pkg/csource/csource_test.go#L157-L169
As a quick unblocking fix I can only suggest to add `manual` as first constraint to match that `# requires: manual` check.

Reply:
The "manual" trick doesn't work. :/

Reply:
Do I need to put it in a dedicated line?

Reply:
I would assume that putting `manual` as the first constraint should work. The check checks for this substring `# requires: manual`.